data/pathfinder/paizo/player_companion/people_of_the_north/potn_abilities_class.lst

```diff
@@ -43,7 +43,7 @@ Rage Powers				KEY:Viking ~ Rage Powers			CATEGORY:Special Ability	TYPE:FighterC
 Class Skills			KEY:Witchguard ~ Class Skills			CATEGORY:Special Ability	TYPE:ClassFeatures.RangerClassFeatures.SpecialQuality.RangerClassSkills																																																																																																																																DESC:A witchguard adds Knowledge (nobility) and Sense Motive to his list of class skills, and removes Handle Animal and Knowledge (dungeoneering) from his list of class skills.																																																																																																																																																																																																																	CSKILL:Climb|TYPE=Craft|Heal|Intimidate|Knowledge (Geography)|Knowledge (Nature)|Knowledge (Nobility)|Perception|TYPE=Profession|Ride|Sense Motive|Spellcraft|Stealth|Survival|Swim			SOURCEPAGE:p.25
-Defend Charge			KEY:Witchguard ~ Defend Charge		CATEGORY:Special Ability	TYPE:ClassFeatures.RangerClassFeatures.SpecialQuality.Extraordinary.RangerHuntersBond																																																																																																																													DESC:At 4th level, a witchguard forms a bond with a spellcaster he has sworn to defend. This bond allows the witchguard to spend a move action to grant an adjacent spellcaster a +%1 dodge bonus to AC and a +2 circumstance bonus on concentration checks. The bonuses last for %2 rounds. The witchguard can use this ability %3 times per day.|(floor(RangerLVL/5)*2)+2|max(1,WIS)|floor((RangerLVL-1)/3)																																																																																																																																																																																																												SOURCEPAGE:p.25
+Defend Charge			KEY:Witchguard ~ Defend Charge		CATEGORY:Special Ability	TYPE:ClassFeatures.RangerClassFeatures.SpecialQuality.Extraordinary.RangerHuntersBond																																																																																																																													DESC:A witchguard forms a bond with a spellcaster he has sworn to defend. This bond allows the witchguard to spend a move action to grant an adjacent spellcaster a +%1 dodge bonus to AC and a +%1 circumstance bonus on concentration checks. The bonuses last for %2 rounds. The witchguard can use this ability %3 times per day.|(RangerLVL/4)*2|max(1,WIS)|floor((RangerLVL-1)/3)																																																																																																																																																																																																																SOURCEPAGE:p.25
```

Comment:
http://paizo.com/pathfinderSociety/rpg/clarifications#RPG

People of the North
Page 25—Replace the third sentence of the Witchguard's Defend Charge ability with the following. "At 8th level and every 4 levels thereafter, these bonuses increase by 2." Replace the second-to-last sentence of the ability with the following. "At 7th level and every 3 levels thereafter, the witchguard can use this ability one additional time per day."